marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/step.sjs

```diff
@@ -31,7 +31,7 @@ class Step {
       this.hubUtils = datahub.hubUtils;
       this.performance = datahub.performance;
     }
-    this.stepTypes = ['ingest', 'mapping', 'custom'];
+    this.stepTypes = ['ingestion', 'mapping', 'custom'];
```

Comment:
@aebadirad ,
Should this include 'mastering' as well ?

Reply:
Yep, it should, good catch